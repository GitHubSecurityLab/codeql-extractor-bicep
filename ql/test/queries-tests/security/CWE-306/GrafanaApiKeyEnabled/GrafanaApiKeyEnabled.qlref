security/CWE-306/GrafanaApiKeyEnabled.ql
