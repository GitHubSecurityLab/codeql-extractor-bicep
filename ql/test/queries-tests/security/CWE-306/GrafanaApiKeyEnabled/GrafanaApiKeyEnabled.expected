| app.bicep:10:13:10:21 | String | Grafana API key feature is enabled, which can increase the attack surface. Consider disabling API keys if not strictly necessary. |
